app/templates/base-en.html

```diff
@@ -87,7 +87,7 @@
 {%- set pageConfig = {
     'title': page_title_value,
     'header': {
-        'logoHref': 'https://www.ons.gov.uk/',
+        'orgLogoHref': domain_url + '/en/start/',
```

Comment:
I think we still want this top logo banner to link to the organisation homepage, rather than our own homepage.